sources/packages/backend/workflow/test/2023-2024/parttime-assessment/parttime-assessment-2023-2024-eligibility-SBSD.e2e-spec.ts

```diff
@@ -0,0 +1,77 @@
+import { PROGRAM_YEAR } from "../constants/program-year.constants";
+import {
+  createFakeConsolidatedPartTimeData,
+  executeAssessment,
+} from "../../test-utils";
+import { InstitutionTypes } from "../../models";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-eligibility-SBSD.`, () => {
+  // Expected and not expected institution types.
+  const EXPECTED_INSTITUTION_TYPES = [
+    InstitutionTypes.BCPublic,
+    InstitutionTypes.BCPrivate,
+  ];
+
+  describe("Should determine SBSD as eligible when total assessment need is greater than or equal to 1, application PD/PPD status is 'yes' and", () => {
```

Comment:
The comment could be explicit about institution type. (BCPublic or BCPrivate)

Reply:
![image](https://github.com/bcgov/SIMS/assets/77353155/6943c59b-fc55-4e2e-a904-3905cb11c0ba)